features/suzie_signs_a_petition.feature

```diff
@@ -76,6 +77,7 @@ Feature: Suzie signs a petition
     When I have already signed the petition with an uppercase email
     And I decide to sign the petition
     And I fill in my details with email "womboidian@wimbledon.com"
+    And I check "Under data protection laws, we need you to confirm you have read our privacy notice and are content for us to use your data in the way described."
```

Comment:
Rather than adding this everywhere, it might be better adding to the `I fill in my details ...` step definition

Reply:
Also I think it will do a partial match of the label - try just "we need you to confirm you have read our privacy notice"